### PR TITLE
show warning when feature hashing is used for multiple featuresets

### PR DESCRIPTION
**Pull request by bndgyawali**
This PR addresses the issue #461.
It shows warning when feature hashing is used on multiple featuresets.
A unit test is added to confirm the behavior.

### CODE REVIEW

File: tests/test_output.py [hunk 1]
```diff
@@ -95,6 +96,7 @@ def tearDown():
                 os.unlink(join(config_dir, cf))
 
         for output_file in (glob(join(output_dir, 'test_{}_*'.format(suffix))) +
+                            glob(join(output_dir, 'test_{}.log'.format(suffix))) +
```
Comment (mulhod) [inline tests/test_output.py]: Is this actually removing all of the outputs or just the `.log` file? You probably need the asterisk here like the others.

Comment (bndgyawali) [reply tests/test_output.py]: Actually, `test_{}_*` was not removing log files which would have format `test_{}.log`. So, I added this extra logic to remove log files.

Comment (desilinguist) [reply tests/test_output.py]: But are there other files named `test_{}` (without the second underscore) that also need to be removed but aren't? Should you change `test_{}.log` to `test_{}.*`?

Comment (bndgyawali) [reply tests/test_output.py]: I did not see any other files other than log. But, if there are, then they should be removed too. Let me check if any.

**Comment by desilinguist** [comment]
Keep it open please

**Comment by desilinguist** [comment]
@pep8speaks suggest diff

**Comment by bndgyawali** [comment]
@desilinguist are you working on this branch? Else, I can confirm @mulhod's comment.

**Comment by desilinguist** [comment]
Nope, I am not working on this branch. I just wanted to make sure the pep8 check bot was working. You can go ahead. Please address the pep8 suggestions as well as @mulhod's comments.

**Comment by bndgyawali** [comment]
This is ready for review.

**Pull request merged**